packages/@aws-cdk/aws-greengrass/package.json

```diff
@@ -55,7 +55,10 @@
     "rosetta:extract": "yarn --silent jsii-rosetta extract"
   },
   "cdk-build": {
-    "cloudformation": "AWS::Greengrass",
+    "cloudformation": [
+      "AWS::Greengrass",
```

Comment:
I remember we regretted putting apigatewayv1 and v2 in the same module and had to revert it after a few months. Let's create a new module for v2

Reply:
OK.